src/_pytest/runner.py

```diff
@@ -2,8 +2,8 @@
 import bdb
 import os
 import sys
-from time import perf_counter
-from time import time
+from time import perf_counter  # Intentionally not `import time` to avoid being
```

Comment:
Weird comment style to fool black into something reasonable. Note that this comment is worth having regardless.